src/main/java/com/bio4j/model/NCBITaxonomyGraph.java

```diff
@@ -8,6 +8,9 @@
 */
 package com.bio4j.model;
 
+import java.util.Arrays;
```

Comment:
nitpicking but better not to import this and use the fqn; this type is not part of any signature here.